rotkehlchen/api/v1/schemas.py

```diff
@@ -173,9 +173,28 @@ class DBPaginationSchema(Schema):
 
 
 class DBOrderBySchema(Schema):
-    # TODO: DBFilters already allow ordering by multiple attributes. Make the API do that too
-    order_by_attribute = fields.String(load_default=None)
-    ascending = fields.Boolean(load_default=False)  # most recent first by default
+    order_by_attribute = fields.List(fields.String(), load_default=None)
+    ascending = fields.List(fields.Boolean(), load_default=None)  # most recent first by default
+
+    @validates_schema
+    def validate_ethtx_query_schema(  # pylint: disable=no-self-use
+            self,
+            data: Dict[str, Any],
+            **_kwargs: Any,
+    ) -> None:
+        if (data['order_by_attribute'] is None) ^ (data['order_by_attribute'] is None):
```

Comment:
Yes, a copy paste error. I'm trying to check that you always provide both because now they are lists. The other option I had in mind was to check the length and create a list of `[False] * len(data['order_by_attribute'])` but felt that this was complicating the logic